allensdk/brain_observatory/behavior/behavior_ophys_session.py

```diff
@@ -179,86 +179,57 @@ def cell_specimen_table(self, value):
         self._cell_specimen_table = value
 
     @property
-    def running_speed(self) -> RunningSpeed:
+    def running_speed(self) -> pd.DataFrame:
         """Running speed of mouse, filtered with 10Hz low pass filter,
-        To get the unfiltered running speed, use `raw_running_speed`.
+
         Returns
         -------
-        allensdk.brain_observatory.running_speed.RunningSpeed
-            NamedTuple with two fields:
-            timestamps : numpy.ndarray
-                Timestamps of running speed data samples
-            values : np.ndarray
-                Running speed of the experimental subject (in cm / s).
+        self._running_speed: pd.DataFrame
+            colums:
+            - timestamps [seconds]
+            - values [cm/s]
+
+        Notes
+        -----
+        To get the unfiltered running speed, use `raw_running_speed`.
         """
         if self._running_speed is None:
-            self._running_speed = self.api.get_running_speed()
+            rs_tuple = self.api.get_running_speed()
+            self._running_speed = pd.DataFrame(
+                    {"timestamps": rs_tuple.timestamps,
+                     "values": rs_tuple.values})
         return self._running_speed
 
     @running_speed.setter
     def running_speed(self, value):
         self._running_speed = value
 
     @property
-    def raw_running_speed(self) -> RunningSpeed:
-        """Running speed of mouse, unfiltered.
+    def raw_running_speed(self) -> pd.DataFrame:
+        """Running speed of mouse, unfiltered
 
         Returns
         -------
-        allensdk.brain_observatory.running_speed.RunningSpeed
-            NamedTuple with two fields:
-            timestamps : numpy.ndarray
-                Timestamps of running speed data samples
-            values : np.ndarray
-                Running speed of the experimental subject (in cm / s).
+        self._running_speed: pd.DataFrame
+            colums:
+            - timestamps [seconds]
+            - values [cm/s]
+
+        Notes
+        -----
+        To get the filtered running speed, use `running_speed`.
         """
         if self._raw_running_speed is None:
-            self._raw_running_speed = self.api.get_running_speed(lowpass=False)
+            rs_tuple = self.api.get_running_speed(lowpass=False)
+            self._raw_running_speed = pd.DataFrame(
+                    {"timestamps": rs_tuple.timestamps,
+                     "values": rs_tuple.values})
         return self._raw_running_speed
 
     @raw_running_speed.setter
     def raw_running_speed(self, value):
         self._raw_running_speed = value
 
-    @property
-    def running_data_df(self) -> pd.DataFrame:
```

Comment:
I think removing these (`running_data_df`, `raw_running_data_df`) will break the NWB api?

https://github.com/AllenInstitute/AllenSDK/blob/8f55db472d58aacc990bedd5bf9f416141e71261/allensdk/brain_observatory/behavior/session_apis/data_io/behavior_nwb_api.py#L60

Reply:
could be, I was counting on some tests failing to show me such things. So far, I have not seen it. I'll look into it.